backend/src/packages/chaiNNer_standard/utility/value/resolutions.py

```diff
@@ -0,0 +1,134 @@
+    ResList.DCI_2KFULL: (2048, 1080),
+    ResList.DCI_4KFULL: (4096, 2160),
+    ResList.DCI_4KFLAT: (3996, 2160),
+    ResList.DCI_4KSCOPE: (4096, 1716),
+    ResList.SQ256: (256, 256),
+    ResList.SQ512: (512, 512),
+    ResList.SQ1024: (1024, 1024),
+    ResList.SQ2048: (2048, 2048),
+    ResList.SQ4096: (4096, 4096),
+    ResList.SQ8192: (8192, 8192),
+    ResList.CUSTOM: (1920, 1080),
+}
+
+assert len(RESOLUTIONS) == len(ResList)
+
+
+@value_group.register(
+    schema_id="chainner:utility:resolutions",
+    name="Resolutions",
+    description="A collection of commonly used image resolution presets.",
+    icon="BsBadgeHd",
+    inputs=[
+        EnumInput(
+            ResList,
+            label="Resolution Presets",
+            default=ResList.HD_1080,
+            option_labels={
+                ResList.HD_1080: "HD 1080P 1920x1080",
+                ResList.NTSC_MPEG2: "SD NTSC MPEG-2 DVD 704x480",
+                ResList.NTSC: "SD NTSC 720x486",
+                ResList.NTSC_DV: "SD NTSC DV 720x480",
+                ResList.PAL: "SD PAL 720x576",
+                ResList.PAL_MPEG2: "SD PAL MPEG-2 DVD 702x576",
+                ResList.HD_720: "HD 720P 1280x720",
+                ResList.UHD_4K: "UHD 4K 3840x2160",
+                ResList.UHD2_8K: "UHD2 8K 7680x4320",
+                ResList.DCI_2KFLAT: "2K DCI FLAT 1998x1080",
+                ResList.DCI_2KSCOPE: "2K DCI SCOPE 2048x858",
+                ResList.DCI_2KFULL: "2K DCI FULL 2048x1080",
+                ResList.DCI_4KFULL: "4K DCI FULL 4096x2160",
+                ResList.DCI_4KFLAT: "4K DCI FLAT 3996x2160",
+                ResList.DCI_4KSCOPE: "4K DCI SCOPE 4096x1716",
+                ResList.SQ256: "Square 256x256",
+                ResList.SQ512: "Square 512x512",
+                ResList.SQ1024: "Square 1024x1024",
+                ResList.SQ2048: "Square 2048x2048",
+                ResList.SQ4096: "Square 4096x4096",
+                ResList.SQ8192: "Square 8192x8192",
+                ResList.CUSTOM: "Custom Resolution",
+            },
+        ),
+        if_enum_group(0, ResList.CUSTOM)(
+            NumberInput(
+                "Width",
+                minimum=1,
+                maximum=None,
+                default=1920,
+                precision=1,
+                controls_step=1,
+                unit="px",
+            ),
+            NumberInput(
+                "Height",
+                minimum=1,
+                maximum=None,
+                default=1080,
+                precision=1,
+                controls_step=1,
+                unit="px",
+            ),
+        ),
+    ],
+    outputs=[
+        NumberOutput("Width", output_type="int(1..)"),
+        NumberOutput("Height", output_type="int(1..)"),
+    ],
+)
+def resolutions_node(
+    resolution_presets: ResList,
+    width: float,
+    height: float,
+) -> tuple[int, int]:
+    if resolution_presets == ResList.CUSTOM:
+        return int(width), int(height)
```

Comment:
```suggestion
    width: int,
    height: int,
) -> tuple[int, int]:
    if resolution_presets == ResList.CUSTOM:
        return width, height
```
You can just type these as ints, since with precision=0 they will be ints